.github/workflows/e2e-manual-icad.yaml

```diff
@@ -27,6 +28,8 @@ on:
           - v0.3.4
           - v0.2.3
           - v0.1.4
+          - pr-139
+          - pr-140
```

Comment:
we'll need to come up with a better way of doing this when we need new versions 🤔